src/ImageSharp/Formats/Jpeg/JpegEncoderCore.cs

```diff
@@ -34,19 +34,24 @@ internal sealed unsafe class JpegEncoderCore : IImageEncoderInternals
         private readonly byte[] buffer = new byte[20];
 
         /// <summary>
-        /// Gets or sets the subsampling method to use.
+        /// The default component id's.
         /// </summary>
-        private JpegSubsample? subsample;
+        private static readonly byte[] DefaultComponentIds = { 1, 2, 3 };
```

Comment:
That way it's a "constant", but it still needs some effort at runtime to use it. See e.g. https://vcsjones.dev/csharp-readonly-span-bytes-static/ for a write-up for this.

These component-id constants are used only at one single place, so I'll write it as suggested in https://github.com/SixLabors/ImageSharp/pull/1734#discussion_r687906455. Yields readable code with good locality, and has best codegen.